test/junit/scala/collection/immutable/LazyListLazinessTest.scala

```diff
@@ -889,6 +889,24 @@ class LazyListLazinessTest {
     assertLazyAll(op)
     assertRepeatedlyLazy(op)
   }
+
+  @Test
+  def sorted_properlyLazy(): Unit = {
+    val sorted = lazyListOp { ll => ll.sorted }
+    val sortBy = lazyListOp { ll => ll.sortBy(identity) }
+    val sortWith = lazyListOp { ll => ll.sortWith(_ < _) }
```

Comment:
I think the following suggestion works, but obviously check

```suggestion
    val sortWith = lazyListOp(_sortWith(_ < _))
```